src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1935,12 +1935,7 @@ private function registerSerializerConfiguration(array $config, ContainerBuilder
                 $container->removeDefinition('serializer.mapping.cache_class_metadata_factory');
             }
 
-            $annotationLoader = new Definition(
-                AnnotationLoader::class,
-                [new Reference('annotation_reader', ContainerInterface::NULL_ON_INVALID_REFERENCE)]
-            );
-
-            $serializerLoaders[] = $annotationLoader;
+            $serializerLoaders[] = new Definition(AttributeLoader::class);
```

Comment:
If you don't pass the annotation reader, annotations will stop working immediately. This is not what we want for 6.4, is it?

Reply:
That's right, I updated this part